src/Elastic.Apm/Model/Transaction.cs

```diff
@@ -117,6 +116,9 @@ ICurrentExecutionSegmentsContainer currentExecutionSegmentsContainer
 		/// </summary>
 		public Context Context => _context.Value;
 
+		[JsonIgnore]
+		internal IConfigSnapshot ConfigSnapshot { get; }
```

Comment:
So, if I understand correctly, the plan is that this is the `ConfigSnapshot` instance that we pass to every other component like `Span` and also this instance is used by things like `TransactionExtensions`, right? If I'm right, then this seems to be a fairly important property and I'd comment it. If I'm wrong, then I'd still comment it, because after looking at the code long enough I got the wrong impression.

Reply:
Done.